deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -337,6 +337,14 @@ def demote_image(request, image_id):
     return HttpResponse("{}", content_type="application/json")
 
 
+def cancel_image_update(request, image_id):
+    try:
+        baseimages_helper.cancel_image_update(request, image_id)
+    except Exception as e:
```

Comment:
In Rodimus I had code to return 400 error for bad requests. We should try to handle those client side errors instead of making everything 500. Same applies to promote and demote.